### ISSUE TITLE
How to make password grant token to get access token and refresh token with multiple providers

### ISSUE THREAD

**Issue by hosseinkarami-dev** [sfelix-martins/passport-multiauth#86]
Hi. I use this package and works great!
but Imagine that I want to sign in with google Auth and check if user exists, make new access token and refresh token for user without requesting to /oauth/token route.
can anyone help me solve my issue considering that we have multiple providers?
Hope Someone solve this.
Thanks alot.

**Comment by sfelix-martins** [comment]
@hkarami1811 you can use personal access tokens. https://github.com/sfelix-martins/passport-multiauth/blob/4.0/README.md#personal-access-tokens

**Comment by hosseinkarami-dev** [comment]
yeah I can use personal access tokens, but i need to get refresh token too. any ideas on how to implement?

**Comment by sfelix-martins** [comment]
I think like the following response https://stackoverflow.com/questions/48755410/how-to-refresh-personal-access-token-programmatically-in-laravel
